[NFR] Persistent toggle Toolbar

lajosbencz opened the pull request:
I really miss the ability to toggle the toolbar... Visible/hidden is stored in a cookie, so it won't reappear after page load.

Here's a working demo: http://static.lazos.me

Having it on the top is also strange (for me at least), but that can be easily fixed with CSS only :]

fabfuel commented on src/Fabfuel/Prophiler/View/sass/screen.scss:
```diff
@@ -2,6 +2,7 @@
 @import "compass/css3";
 @import "compass/utilities/general/clearfix";
 
+$c-background: #fff;
```
In your demo, the background colors are not aligned. The button's background color is #fff, but the toolbar's is #f8f8f8

fabfuel commented on src/Fabfuel/Prophiler/View/toolbar.phtml:
```diff
@@ -89,5 +97,27 @@ use Fabfuel\Prophiler\Toolbar\Formatter\BenchmarkFormatter;
+        // Move toggle event to delegate handler
+        $('body')
+            .on('click', '#prophiler-toggle', function() {
+                $(this).find('.fa').toggleClass('fa-angle-up fa-angle-down');
```
We can not assume, the user's project is using Font Awesome. 
The toggle button needs to work/be visible without it as well.

fabfuel commented:
Thanks for your pull request! I really like it, but please have a look at the comments.

Best
Fabian

lajosbencz commented:
Thanks for the feedback Fabian :]
I've changed it according to your comments! I also reverted to the body margin-top placeholder, because the extra div haven't worked out after all... DOM order could be messed up in other projects.

The only thing that still bothers me is the jumpy loading if the toolbar is turned on when the page loads...  I tried moving the init code outside document.load with pure JS, but it didn't help either :( Any tips for minimizing that annoying jumpiness? Demo page is still online with the new mods.

harikt commented:
I was browsing to have some sort of hide / show functionality. Probably this is the PR I am looking for.

sergeyklay commented:
+1
